apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -110,9 +111,7 @@ const BookingPage = (props: BookingPageProps) => {
 
   const rescheduleUid = router.query.rescheduleUid as string;
   const { isReady, Theme } = useTheme(props.profile.theme);
-
   const date = asStringOrNull(router.query.date);
-  const timeFormat = asStringOrNull(router.query.clock) === "24h" ? "H:mm" : "h:mma";
```

Comment:
Where was this router.query.clock being used, need to investigate as [Chesterton's fence ](https://fs.blog/chestertons-fence/)may apply here (look for it in the codebase)